### PR TITLE
bot API 7.10

### PR DESCRIPTION
**Pull request by coder2020official**
- [x] Added updates about purchased paid media, represented by the class [PaidMediaPurchased](https://core.telegram.org/bots/api#paidmediapurchased) and the field purchased_paid_media in the class [Update](https://core.telegram.org/bots/api#update).
- [x] Added the ability to specify a payload in [sendPaidMedia](https://core.telegram.org/bots/api#sendpaidmedia) that is received back by the bot in [TransactionPartnerUser](https://core.telegram.org/bots/api#transactionpartneruser) and purchased_paid_media updates.
- [x] Added the field prize_star_count to the classes [GiveawayCreated](https://core.telegram.org/bots/api#giveawaycreated), [Giveaway](https://core.telegram.org/bots/api#giveaway), [GiveawayWinners](https://core.telegram.org/bots/api#giveawaywinners) and [ChatBoostSourceGiveaway](https://core.telegram.org/bots/api#chatboostsourcegiveaway).
- [x] Added the field is_star_giveaway to the class [GiveawayCompleted](https://core.telegram.org/bots/api#giveawaycompleted).
- [ ] Added the field SecondaryButton to the class [WebApp](https://core.telegram.org/bots/webapps#initializing-mini-apps).
- [ ] Added the event secondaryButtonClicked for Mini Apps.
- [ ] Added the field bottomBarColor and the method setBottomBarColor to the class [WebApp](https://core.telegram.org/bots/webapps#initializing-mini-apps).
- [ ] Added the field bottom_bar_bg_color to the class [ThemeParams](https://core.telegram.org/bots/webapps#themeparams).

### CODE REVIEW

**Comment by coder2020official** [comment]
Done. All others are related to webapps.
@badiboy review plz

**Comment by Badiboy** [comment]
Got it

**Comment by Badiboy** [comment]
@coder2020official Sorry, hard week

File: telebot/apihelper.py [hunk 1]
```diff
@@ -530,7 +530,7 @@ def send_paid_media(
         token, chat_id, star_count, media,
         caption=None, parse_mode=None, caption_entities=None, show_caption_above_media=None,
         disable_notification=None, protect_content=None, reply_parameters=None, reply_markup=None,
-        business_connection_id=None):
+        business_connection_id=None, media_payload=None):
```
Comment (Badiboy) [inline telebot/apihelper.py:533]: Why parameter named "media_payload", but not "payload"?

I propose to set it to "payload" to keep naming consistency, and rename internal "payload" to something else. May be "_payload".

File: telebot/asyncio_helper.py [hunk 2]
```diff
@@ -520,7 +520,7 @@ async def send_paid_media(
         token, chat_id, star_count, media,
         caption=None, parse_mode=None, caption_entities=None, show_caption_above_media=None,
         disable_notification=None, protect_content=None, reply_parameters=None, reply_markup=None,
-        business_connection_id=None):
+        business_connection_id=None, media_payload=None):
```
Comment (Badiboy) [inline telebot/asyncio_helper.py:523]: The same here.

File: telebot/types.py [hunk 3]
```diff
@@ -9479,11 +9499,12 @@ def de_json(cls, json_string):
         obj['user'] = User.de_json(obj.get('user'))
         return cls(**obj)
 
-    def __init__(self, source, giveaway_message_id, user=None, is_unclaimed=None, **kwargs):
+    def __init__(self, source, giveaway_message_id, user=None, is_unclaimed=None, prize_star_count=None, **kwargs):
         self.source: str = source
         self.giveaway_message_id: int = giveaway_message_id
         self.user: User = user
```
Comment (Badiboy) [inline telebot/types.py:9505]: user should be Optional

File: telebot/types.py [hunk 4]
```diff
@@ -9479,11 +9499,12 @@ def de_json(cls, json_string):
...
         self.is_unclaimed: bool = is_unclaimed
```
Comment (Badiboy) [inline telebot/types.py:9506]: is_unclaimed should be Optional

**Comment by Badiboy** [comment]
Also need to add "purchased_paid_media" to util.update_types.

And please, bump the version also )

**Comment by coder2020official** [comment]
Version is 4.23.0 which is still unreleased, no need for bumping

**Comment by Badiboy** [comment]
I propose to push a release.

**Pull request merged**